be/src/vec/functions/function_cast.h

```diff
@@ -428,7 +429,6 @@ struct ConvertImpl {
                     for (size_t i = 0; i < size; ++i) {
                         (*vec_null_map_to)[i] = !TimeCast::try_parse_time(
                                 vec_from[i], vec_to[i], context->state()->timezone_obj());
-                        vec_to[i] *= (1000 * 1000);
```

Comment:
why delete the code？

Reply:
TimeCast::try_parse_time handles this part.